### PR TITLE
Release 3.0.1

### PR DESCRIPTION
**Pull request by Jkovarik**
**Summary:** Summary of changes

Addresses [CUMULUS-1](https://bugs.earthdata.nasa.gov/browse/CUMULUS-1)

## Changes

* Backport of #1941, #1948, #1931, #1905, #1897 to Release 3.0.x baseline

## PR Checklist

- [ ] Update CHANGELOG
- [ ] Unit tests
- [ ] Adhoc testing
- [ ] Integration tests

### CODE REVIEW

File: tasks/hello-world/package.json [hunk 1]
```diff
@@ -33,6 +33,6 @@
   "dependencies": {
     "@cumulus/aws-client": "3.0.0",
     "@cumulus/common": "3.0.0",
-    "@cumulus/cumulus-message-adapter-js": "1.3.1"
+    "@cumulus/cumulus-message-adapter-js": "1.3.2"
```
Comment (kkelly51) [inline tasks/hello-world/package.json:36]: here, 1.3.2 ?

Comment (Jkovarik) [reply tasks/hello-world/package.json]: Good catch, ty! d707286

**Review by kkelly51 — APPROVED** [review]
👍

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,11 +6,24 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 ## [Unreleased]
 
+- **CUMULUS-2203**
```
Comment (markdboyd) [inline CHANGELOG.md:9]: Seems like these changes should be under a header for 3.0.1?

Comment (Jkovarik) [reply CHANGELOG.md:9]: That's in the release PR: https://github.com/nasa/cumulus/pull/1952

Comment (Jkovarik) [reply CHANGELOG.md:9]: Note 2200 should have been in the 3.0.0 release info, I'm going to update the master CL as well as the release notes.

**Pull request merged**